examples/hotrod/services/frontend/web_assets/index.html

```diff
@@ -3,7 +3,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0">
   <head>
     <title>HotROD - Rides On Demand</title>
-    <script src="https://code.jquery.com/jquery-3.1.1.min.js"></script>
+    <script src="./jquery-3.1.1.min.js"></script>
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
```

Comment:
what about the Bootstrap CSS and JS? Wouldn't they present similar issues?

Reply:
I have tested for load two bootstrap files locally; It will have some impact on the page display; moreover, this address can be accessed in China, so I don't think it is necessary to modify the loading of these two files.

Reply:
ok, works for me